packages/taco/src/conditions/const.ts

```diff
@@ -11,5 +11,6 @@ export const SUPPORTED_CHAIN_IDS = [
   ChainId.POLYGON,
   ChainId.MUMBAI,
   ChainId.GOERLI,
+  ChainId.SEPOLIA,
```

Comment:
Can we add a test to ensure this list is kept up-to-date? We did something analogous in `nucphyer/nucypher` - https://github.com/nucypher/nucypher/commit/c9b417271a9136188ac0231c22116f61e2655aab.